src/main/java/com/geronimo/model/User.java

```diff
@@ -1,15 +1,38 @@
 package com.geronimo.model;
 
 import lombok.Data;
+import lombok.ToString;
 
-import javax.persistence.Entity;
-import javax.persistence.Id;
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.*;
 
-@Entity(name = "users")
+@Entity
 @Data
+@ToString(exclude = {"following", "messages", "followers"})
+@Table(name = "users")
 public class User {
     @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
     private Long id;
+
+    @NotNull
     private String username;
+
+    @NotNull
     private String password;
+
+    @OneToOne(cascade=CascadeType.ALL)
+    @NotNull
+    private Profile profile = new Profile();
```

Comment:
Make this class embedded, we don't need to create a new table for profile. Agreed? 

If you make this embedded, fields from Profile class will be extracted to the User table.

Reply:
When I started coding it, it was embedded, then I thought it was a bit too messy to keep it all in one table.